### ISSUE TITLE
Screenfetch won't install on Shrine 5.05

### ISSUE THREAD

**Issue by pizza-eater** [minexew/Shrine#49]
I can't download screenfetch from the repository. There is a problem with version mismatch.

![image](https://user-images.githubusercontent.com/46658420/97237949-47925500-17f9-11eb-90c8-60b5934444a4.png)

**Comment by minexew** [comment]
Hi, thanks for reaching out.

I'm afraid that the error is correct, as Screenfetch really will not work on the latest version, because it needs BMP format support which was removed from TempleOS upstream at some point and I have never gotten around to putting it back in (see issue #8)

**Comment by pizza-eater** [comment]
Sorry to write here. I didn't find your mail or discord server. How is the distribution development progressing?

**Comment by minexew** [comment]
Hey, no problem.

I do have a sort-of roadmap drafted, and there is a lot that I'd like to do on Shrine. However, recently I have been focusing my energy on things that I perceive as more _meaningful_. In part it's because I find the majority of the "community" around TempleOS despicable, but it's only one of many reasons. Another one would be the lack of other people coding in TempleOS and writing about it.

In summary, there is still chance for Shrine 6 before the end of 2020, but don't hold your breath :) Perhaps you would appreciate [ZenithOS](https://github.com/TomAwezome/ZenithOS) which seems to have more active development?

**Comment by pizza-eater** [comment]
Of course, I would like to see Terry's legacy develop in different ways. I am very impressed with this kind of work, so I am very happy when I see that people are improving this legendary OS. I know about the community. What can I do? Terry became popular on the Internet mostly because of his negative side, which is quite sad. I hope you don't give up on this project.

**Comment by minexew** [comment]
@lerhyd Hi, could you please re-try now? I have updated the package to bundle the BMP code (this way it will also work on plain TempleOS).

**Comment by pizza-eater** [comment]
If I use the correct version of Shrine, everything is fine.
![image](https://user-images.githubusercontent.com/46658420/98998517-4f672e80-2547-11eb-85e5-ccea4272c478.png)

**Comment by minexew** [comment]
Nice!

Since you say "If I use the correct version of Shrine...", is there a specific version of Shrine that you found _not_ to work?

If you're referring to the errors in the right pane of the screen, the following commands should work:
```c
#include "/Apps/Screenfetch/Screenfetch"
Screenfetch;
```
that's just how things work in the TempleOS land :)
